pkgs/top-level/all-packages.nix

```diff
@@ -645,6 +645,8 @@ in
 
   altermime = callPackage ../tools/networking/altermime {};
 
+  alttab = callPackage ../tools/X11/alttab { };
```

Comment:
I feel this isn't the right place but the file is in such a mess I can't tell where the right place is

Reply:
I assumed the file was ordered alphabetically... With some exceptions anyway

Reply:
Its meant to be in sections and then alphabetically in those sections I think

Reply:
I wouldn't worry about it for now